components/collective-navbar/index.js

```diff
@@ -213,7 +213,7 @@ const getHasContribute = (collective, sections, isAdmin) => {
   );
 };
 
-const getDefaultCallsToActions = (collective, sections, isAdmin, isHostAdmin, isRoot) => {
+const getDefaultCallsToActions = (collective, sections, isAdmin, isHostAdmin, isRoot, LoggedInUser) => {
```

Comment:
:ice_cream: Since the `isRoot` passed here directly comes from `LoggedInUser` and is not used in the previous function, taking just the LoggedInUser would simplify

```suggestion
const getDefaultCallsToActions = (collective, sections, isAdmin, isHostAdmin, LoggedInUser) => {
```

Reply:
Thanks. Done. 👍🏽